cleanlab/count.py

```diff
@@ -199,6 +202,44 @@ def calibrate_confident_joint(confident_joint, labels, *, multi_label=False) ->
     return round_preserving_row_totals(calibrated_cj)
 
 
+def _calibrate_confident_joint_multilabel(confident_joint: np.ndarray, labels: list) -> np.ndarray:
```

Comment:
Note the docstring for public `calibrate_confident_joint` function seems a little off now that we've updated multilabel. 
https://github.com/cleanlab/cleanlab/blob/9f52e346ea1cf65c8a7425d276b0e29b2076fb4a/cleanlab/count.py#L144

Specifically this part:
```
multi_label : bool, optional
      If ``True``, labels should be an iterable (e.g. list) of iterables, containing a
      list of labels for each example, instead of just a single label.
      The multi-label setting supports classification tasks where an example has 1 or more labels.
      Example of a multi-labeled `labels` input: ``[[0,1], [1], [0,2], [0,1,2], [0], [1], ...]``.
      The major difference in how this is calibrated versus single-label is that
      the total number of errors considered is based on the number of labels,
      not the number of examples. So, the calibrated `confident_joint` will sum
      to the number of total labels.
```

Also the docstring for `confident_joint` arg does not mention the new multilabel shape at all.

Reply:
Same issue for `confident_joint` arg in `estimate_joint()` documentation. Nothing is mentioned about how it should look for multilabel setting.

Reply:
Fixed : https://github.com/aditya1503/cleanlab/blob/8b088504e89eb7c055de1b2ae9e9405f9ff28329/cleanlab/count.py#L179